playbooks/ansible-security-trendmicro-deepsec-appliance/pre.yaml

```diff
@@ -1,15 +1,9 @@
 ---
-- hosts: controller
+- hosts: appliance*
+  gather_facts: false
+  vars:
+    trendmicro_deepsec_api_key: '{{ zuul.executor.work_root }}'
   tasks:
-    - name: Ensure tox
+    - name: Setup TM instance silently
       include_role:
-        name: ensure-tox
```

Comment:
for now, revert this. and in the follow up PR we can remove and address ARA failures